runtime/cudaq/platform/default/rest_server/helpers/RestRemoteClient.cpp

```diff
@@ -371,7 +392,15 @@ class NvcfRuntimeClient : public RemoteRestRuntimeClient {
     {
       const auto funcIdIter = configs.find("function-id");
       if (funcIdIter != configs.end()) {
+        // User overrides a specific function Id.
         m_functionId = funcIdIter->second;
+        if (m_logLevel > LogLevel::None) {
+          // Print out the configuration
+          printf("%s\n",
+                 fmt::format("Submitting jobs to NVQC using function Id {}.",
+                             m_functionId)
+                     .c_str());
```

Comment:
Why not use `fmt::print()` instead? (Same question for other usages.)

Reply:
I guess because `fmt::print` is not very widely used in our codebase yet (it should be, I think) so my hands just typed `printf` :-)
Converted all `printf` usages in this file to `fmt::print`.